protos/perfetto/config/chrome/BUILD.gn

```diff
@@ -21,4 +21,7 @@ perfetto_proto_library("scenario_@TYPE@") {
   sources = [ "scenario_config.proto" ]
   generate_descriptor = "scenario_config.descriptor"
   descriptor_root_source = "scenario_config.proto"
+  generator_visibility = [
```

Comment:
This needs to be guarded with build_with_chromium.

Reply:
If we want to avoid having path of downstream project, we can set `"*"` instead to make generator action public.

Reply:
That would be ideal yes.

Reply:
OK, updated to use `"*"`.